### DISCUSSION TITLE
Bi-weekly accessibility meeting

### DISCUSSION THREAD

**Discussion by bl-aire** [jupyter/accessibility#117] [Meeting]
Hello,

For about two consecutive meetings I have tried joining but the link below did not go through. Am I using the right link? Or it was changed?

<img width="1280" alt="Screenshot 2023-02-13 at 10 20 34" src="https://user-images.githubusercontent.com/84882370/218432906-3e2f9779-11a0-40ed-88af-23769f8486ee.png">

**Comment by trallard** [comment]
Hey @bl-aire, thanks for opening this discussion, I just had a quick check, and the Zoom link is the correct one. 
Do you have any more details on the type of error you see? - Any additional information you have will be useful to identify the source of the error and ensure anyone wanting to join has the correct information for this.


There is also the [Jupyter community calls calendar](https://docs.jupyter.org/en/latest/community/content-community.html), which you can use to track the meetings and add these directly to your calendar.

**Reply by bl-aire** [reply]
Hi @trallard, 

The error my browser displays is "Safari cannot open the page because the address is invalid". I downloaded Zoom's mobile app and was able to access the link so I'll use that henceforth.

And I have added the Jupyter community calls calendar to mine.

Thank you.
